cmd/earthly/main.go

```diff
@@ -1053,6 +1058,105 @@ Set up a whole custom git repository for a server called example.com, using a si
 				},
 			},
 		},
+		{
+			Name:    "satellite",
+			Aliases: []string{"satellites"},
+			Usage: "Launch and use a Satellite runner as remote backend for Earthly builds.\n" +
+				"	Satellites can be used to optimize and share cache between multiple builds and users,\n" +
+				"	as well as run builds in native architectures independent of where the Earthly client is invoked.\n" +
+				"	Note: this feature is currently experimental.\n" +
```

Comment:
Since it's experimental, should we include a `Hidden:      true` field?

Reply:
our beta testers can add an `export EARTHLY_SHOW_HIDDEN=1` to enable this help text.